### DISCUSSION TITLE
[QUESTION] Location of action files

### DISCUSSION THREAD

**Discussion by dglava** [puddletag/puddletag#632] [General]
Is there a reason why action files get saved under `~/.local/share/applications/Puddletag/actions` and not in the config directory? It's just a minor inconvenience when backing up settings, but I was wondering if there's a specific reasoning behind it, and if not, if it could be moved to the configuration directory?

**Comment by samuraibrian** [comment]
Or ~/.local/share/plugins as it happens to be on my Ubuntu 18.04 system... :-) (Puddletag 1.2.0)

**Comment by sandrotosi** [comment]
I think this is a conscious decision: it is set on [this line of code](https://github.com/puddletag/puddletag/blob/master/source/puddlestuff/constants.py#L58) which originated from [this commit](https://github.com/puddletag/puddletag/commit/67bf3138a7e9ed2cac37f3008c4e6f2abb022371); it refers to the XDG spec, which is available [here](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), and it reads 

> $XDG_DATA_HOME defines the base directory relative to which user-specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used._
